docs/getting-started.md

```diff
@@ -50,6 +50,35 @@ checked-in [deployment yaml](/build/yamls/antrea.yml):
 kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea.yml
 ```
 
+### Deploying Antrea on a Cluster with Existing CNI
+
+Ideally Antrea should only be deployed on a new cluster, which is not already
+running a CNI network plugin. If you need to migrate your existing cluster from
+another CNI to Antrea, you will need to do the following:
+ * Delete previous CNI, including all resources (K8s objects, iptables rules,
+ interfaces, ...) created by that CNI.
+ * Deploy Antrea.
+ * Restart all Pods (you may use `kubectl drain` to drain each Node or restart
+ all your Nodes) in order for Antrea to set-up networking for them.
+
+While this is in-progress, networking will be disrupted in your cluster. After
+deleting the previous CNI, existing Pods may not be reachable anymore.
+
+For example, when migrating from Flannel to Antrea, you will need to do the
+following:
+1. Delete Flannel with `kubectl delete -f <path to your Flannel YAML manifest>`.
+2. Delete Flannel bridge with `ip link delete flannel.1 && ip link delete
+flannel cni0` **on each Node**.
+3. [Deploy Antrea](#installation).
+4. Drain and uncordon Nodes one-by-one. For each Node, run `kubectl drain
```

Comment:
drain does not provide any real value (compared to just restarting the Nodes) here since old Pods are not reachable anymore... maybe I should just say that all Nodes must be restarted?